### ISSUE TITLE
Status of TL-WN821N V5

### ISSUE THREAD

**Issue by callegar** [ZeeRooo/RTL8192EU#7]
Driver seems to support the TL-WN821N V5 too.

There is an issue, though which is probably in common with the TL-WN821N V6.

If you load the module and connect, the connection is fine.
If you ask network manager to unconnect and then you try to connect again, it does not work anymore.

**Comment by brij2001** [comment]
does it work with monitor mode though? Normal Wifi connection is works flawlessly but can you check compatibility with airmon?

**Comment by callegar** [comment]
No, sorry for the noise, the driver is broken for the TL-WN821N V5. It connects for the first time after plugging, but if you disconnect and try to reconnect, it hangs.

**Comment by 4lch3my** [comment]
Any Update on this maybe?
